portfolio/src/main/webapp/style.css

```diff
@@ -1,16 +1,602 @@
+*, *:before,
```

Comment:
this is selecting all elements on the page, the `:before` selector for all elements and the `:after` elements on the page. Can I ask what you're trying to do here? I imagine we can get rid of this and work to be a little more targetted than selecting all elements on the page 3 times, this is fairly inefficient.

Put another way, remove this and identify what's changed with your page. We can work to fix this problems.

Reply:
I had some issues with getting the footer to stay at the bottom of the page content so I thought that would help. Doesn't make any difference - removed it!